src/docs/spec/page-header.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { Subheader } from '../docs-components/typography'
+
+const Header = props => (
+  <div>
+    <Subheader>{props.description}</Subheader>
+  </div>
```

Comment:
Can we place the description in a `<Subheader>` like in the previous version of this file? I meant for it go here 😉